### DISCUSSION TITLE
Scrum Prep Milestone 2 - Week 3

### DISCUSSION THREAD

**Discussion by Ar2691** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#112] [Scrum Preps]
Oct 4 - Oct 11

**Comment by christianmedina-dev** [comment]
Login Team here. Developed the Mock Up for the Sign Up Form in Figma and attended group meeting.

**Comment by cristianMarcial** [comment]
Curriculum restructuring team leader here. This week we conducted research on how to parse a scraped website in order to obtain the UPRM course catalog for this semester. Then, we defined the tasks related to creating the modules that are responsible for uploading the data obtained by parsing the scraped data.

**Comment by keisanabria** [comment]
Created the minute of scrum meeting during October 4, 2024. Read the professor's feedback on Milestone 1 Documentation submission. Created an Issue that required to create the minutes and submitted them in the comments of the Issue. After this, the managers approved of the issue. Was assigned as Timekeeper of future scrum and general meetings of RegiUPR. An Issue was created for the latter with its description, and it got the approval of the managers and team leader; later on, it was closed successfully.

**Comment by Gian-Go** [comment]
Created and worked on two documentation tasks related to modifications to the document based on given feedback from milestone 1 as well as additions relevant to the past sprint for milestone 2.

**Comment by PedroRodz** [comment]
Closed a task. Read peers’ tasks to stay up to date on what is happening.

**Comment by J3SSY-ANDU** [comment]
Created a design for the mock-up college planning feature design. Completed my lecture topic task about unit testing of the students and student_courses database tables crud functions.

**Comment by nicolle-feliciano** [comment]
Created and completed a task to summarize the professor’s feedback on Milestone 1’s documentation.

**Comment by cristianramos9** [comment]
Conduct research on web scraping and parsing the downloaded page. Specifically a table with courses available.

**Comment by carlospepin23** [comment]
- Completed the task of integrating semester and course planning into the database schema.

**Comment by Carloscaban** [comment]
Part of the login screen team. Worked on password recovery tasks. Implemented various features.
